views/index.html

```diff
@@ -306,6 +306,20 @@
 {{/unless}}
 
 <script src="{{static}}/js/vendor/jquery-1.11.0.min.js"></script>
+<script>
+  // Load libraries from a server config file
+  // This method is used so we can both expand vars, and support <script> in the json strings
+  window.libraries = window.libraries || [];
+  $.ajax({
```

Comment:
Moved this to async true as the last script run. It can't be a directly loaded as a var because javascript reads </script> anywhere (yes, even in a string) to end a script ("snippet" contains this).